pkg/sql/opt/testutils/opttester/testdata/bundle/stats-defaultdb.public.a.sql

```diff
@@ -0,0 +1,1240 @@
+ALTER TABLE public.a INJECT STATISTICS '[
```

Comment:
nit: You can probably use a smaller set of stats for testing purposes.

Reply:
:+1: